data/human/human missions.txt

```diff
@@ -6767,3 +6767,123 @@ mission "Hogshead's Dilemma II"
+			`	He sighs and says, "Card, please." He holds out a silver tray and awaits your calling card.`
+			choice
+				`	(Explain you don't have a calling card.)`
+					goto nocard
+				`	(Place your card on the tray.)`
+					to display
+						"net worth" > 20000000
+
+			`	The butler's eyebrows rise in the slightest hint of surprise when you place your card on the tray. He promptly turns and disappears into the overlarge house. You, Caesar, and the footman all wait for half an hour before a woman arrives at the front entrance, wearing an exceedingly complicated gown.`
+				goto boolinbrook
+
+			label nocard
+			`	The butler smiles smugly. "How unfortunate. In that case, I have no choice but to bring you in through the servant's entrance. It's only proper."`
+			`	Instead of taking you up the wide stairs to the beautifully inscribed wooden doors, he ushers you around the side to a small plain door. You are left there with the footman to wait.`
+			`	Three quarters of an hour later, a woman wearing an exceedingly impractical dress arrives.`
+
+			label boolinbrook
+			`	"Mr. Boolinbrook says you've brought back my darling See-See?" She looks down her nose at you and waits for your response.`
+			choice
+				`	"Your dog is safe and sound with me."`
+				`	"I found Caesar alone, out in a park on Calda."`
+				`	"Here he is. You're welcome."`
+
+			`	Before you can respond, Caesar starts yapping happily and runs up to the lady.`
+			`	She snatches up the dog and draws it to her bosom. "Oh my darling See-See! It has been most dreadful waiting for news of you!"`
+			`	After a moment she turns her attention back to you, looking slightly softer than before. "See-See tires of the parks here, and so every Tuesday I send him to Calda for a change of pace. It's good for his constitution, you know. Anyway, the horrid man who normally ferries our dogs on these little excursions had some sort of episode. A stroke, or a heart attack - you get the idea. He had an assistant return the dogs, but in the confusion, they forgot my little See-See. I was sure some gauche tourist would have snatched him up and eaten him by now."`
+			`	She turns her attention back to the dog and speaks to him in baby talk. "But, instead, this gallant captain saved you!"`
+			`	The lady snaps her fingers, and the butler is immediately at her side. "We must make this person whole. I can't be perceived as owing favors to vagabonds. Imagine the talk!`
+			`	The butler quickly procures a credit chip and shows it to her, accepting a curt nod for approval. "I realize that this is probably a fortune to you," she says. "I recommend you find a financial advisor and invest it well."`
+			`	With that, she turns and leaves; the butler likewise disappears, and you're escorted by the footman back to your vehicle.`
+			`	You look down at the credit chip in your hand and read its value: <payment>.`
+
+
+mission "Lost Dog: Homeward Bound"
+	to offer
+		random < 3
+		has "Lost Dog: declined"
```

Comment:
I haven't tested this yet, but it seems this mission would trigger even if I never went to the park?